custom/v3/note/util/src/main/java/org/apache/fineract/v3/note/util/NoteUtilityMethods.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.v3.note.util;
+
+import org.apache.fineract.portfolio.note.domain.NoteType;
+import org.apache.fineract.portfolio.note.exception.NoteResourceNotSupportedException;
+
+public final class NoteUtilityMethods {
+
+    private NoteUtilityMethods() {}
+
+    public static NoteType getNoteType(final String resourceType) {
```

Comment:
How many times do you use this class? I bet only 2-3 times... and only in the API layer... doesn't seem worth it and again makes readers jump back and forth when they read the controller classes. If you are not saving a hundred lines of code (you won't) then please inline.

Reply:
Noted.